pkg/skaffold/schema/v1beta13/upgrade_test.go

```diff
@@ -35,6 +35,13 @@ build:
   - image: gcr.io/k8s-skaffold/bazel
     bazel:
       target: //mytarget
+  - image: gcr.io/k8s-skaffold/jib-maven
+    jibMaven:
+      args: ['-v']
```

Comment:
should add in a test with `module` -> `project` in jibMaven?

Reply:
👍